server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -925,9 +922,6 @@ private RepositoryData getRepositoryData(long indexGen) {
                 repositoryData = RepositoryData.snapshotsFromXContent(parser, indexGen);
             }
             return repositoryData;
-        } catch (NoSuchFileException ex) {
```

Comment:
nit: while we are at it, there are unneeded `Long.toString()` and local variable `repositoryData` usages

Reply:
I can't find it right now, but others disagreed with removing the `Long.toString` in another PR :( But yea let's clean up the redundant locals :)